sst/.gitignore

```diff
@@ -0,0 +1,15 @@
+# dependencies
+node_modules
+
+# sst
+.sst
+.build
+
+# misc
+.DS_Store
+
+# local env files
+.env*.local
+
+# file for setting secrets via CLI
```

Comment:
Note for writing README:

Example command is `res=$(cat secrets.txt) && npx sst secrets set POSTMAN_API_KEY $res`